microcosm_pubsub/fields.py

```diff
@@ -1,9 +1,11 @@
 """
 Custom fields.
+
 """
-from marshmallow.fields import Field, ValidationError
 from uuid import UUID
 
+from marshmallow.fields import Field, ValidationError
+
```

Comment:
Out of curiosity since you've changed the order of the imports - does it matter here?